about.html

```diff
@@ -0,0 +1,135 @@
+                        <a class="nav-link" href="#">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Join</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link aseel-name p-1">Aseel Afghanistan</a>
+                    </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <!-- HEADER SECTION END HERE -->
+    <!-- FIRST SECTION START HERE -->
+    <section class="first-section">
+        <div class="container">
+            <div class="contant">
+                <div class="first-color text-center p-4">
+                    <h2>Aseel | Buy Good, Do Good</h2>
+                </div>
+                <div class="about-p p-4 m-1">
+                    <p>Aseel Do Good is the most reliable and pioneering platform working to fight the humanitarian crisis in Afghanistan – the first country of operation. It allows anyone who wants to assist Afghan families in need to Purchase Emergency Packages, Sponsor a Family, Start a Campaign, and Become a Volunteer. We partner with organizations that help us provide vulnerable families and individuals with basic food and shelter items. Hence, supporting them to build a brighter and more sustainable tomorrow for themselves.</p>
+                </div>
+                <div class="d-flex text-center flex-column p-4">
+                    <p>Aseel Buy Good is an online marketplace that links aspiring makers with buyers all over the world who value unique, artistic handmade products.</p>
+                    <h6>aseel@aseelapp.com</h6>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- FIRST SECTION END HERE -->
+    <!-- SECOND SECTION START HERE -->
+    <section class="second-section-about fourth-color">
+        <div class="container">
+            <div class="content d-flex flex-column text-center p-5">
+                <div class="title">
+                    <h3>Aseel | Buy Good, Do Good</h3>
+                </div>
+                <div class="w-100 d-flex justify-content-center">
+                    <p class="w-50">Aseel is a tech-based startup company providing a digital platform that allows artisans to sell their antique handcrafted products and provide humanitarian aid to and from anywhere in the world with complete transparency. We believe in the practical use of the skills to build the positive change the businesses and communities need. The easy and user-friendly platform of Aseel supports thousands of makers by connecting them to global buyers. Transparency and privacy have always been the core values of our team. We are committed to supporting our clients through our secured and protected customer service page anytime.</p>
```

Comment:
- [ ] It will be better to give this content `width: 100%` in the mobile version to make your content take the width of container and to be aligned with design guidelines.